backoffice/backoffice/authors/airflow_utils.py

```diff
@@ -134,24 +123,19 @@ def delete_workflow_dag(dag_id, workflow_id):
     """Delete dag run.
 
     :param dag_id: dag to be removed
-    :param workflow_id: id of workflow whoose dag execution should be deleted
+    :param workflow_id: id of workflow whose dag execution should be deleted
     :returns: request response
     """
 
     url = f"{AIRFLOW_BASE_URL}/api/v1/dags/{dag_id}/dagRuns/{str(workflow_id)}"
-    try:
-        logger.info(
-            "Deleting dag Failed Tasks of DAG %s with no data and %s",
-            dag_id,
-            url,
-        )
-        response = requests.delete(url, headers=AIRFLOW_HEADERS)
-        response.raise_for_status()
-        return JsonResponse({"message": "Successfully deleted DAG"})
-    except RequestException:
-        return JsonResponse(
-            {"error": "Failed to delete DAG"}, status=status.HTTP_424_FAILED_DEPENDENCY
-        )
+    logger.info(
+        "Deleting dag Failed Tasks of DAG %s with no data and %s",
+        dag_id,
+        url,
+    )
+    response = requests.delete(url, headers=AIRFLOW_HEADERS)
+    response.raise_for_status()
+    return response
```

Comment:
I’m quite skeptical about returning the entire response object. In our case, we only need the `data` and `code`. Additionally, there’s a risk that the requests API might change in the future, requiring us to modify all the methods instead of just one place here.